test/prop_statem.erl

```diff
@@ -275,6 +274,11 @@ teardown(_) ->
 %%==============================================================================
 cleanup() ->
   catch disconnect(),
+  %% Restart the server, since though the client disconnects the
+  %% server keeps its state.
+  application:stop(ranch),
```

Comment:
We should add ranch to the `.app.src`.

Reply:
It is added, but stopping an application doens't stop it's dependecy applications.